src/util/validation.js

```diff
@@ -62,6 +62,9 @@ const sharedQuerySchema = Joi.object({
   'screenshot.clip.width': Joi.number(),
   'screenshot.clip.height': Joi.number(),
   'screenshot.omitBackground': Joi.boolean(),
+  // 'request.headers': Joi.object(), // any way to send an object?
```

Comment:
Not sure if there are any clever workarounds here. Could optionally be a json string that gets decoded? I think https://github.com/alvarcarto/url-to-pdf-api/pull/78 has this same issue.